ckan/lib/search/jobs.py

```diff
@@ -0,0 +1,70 @@
+# encoding: utf-8
+
+# (canada fork only): background search index rebuilding
+#TODO: upstream contrib!!
+
+import json
+import datetime
+from rq import get_current_job
+
+import ckan.logic as logic
+import ckan.lib.search as search
+import ckan.model as model
+
+from ckan.plugins import toolkit
+
+from logging import getLogger
+log = getLogger(__name__)
+
+
+def reindex(package_ids=None, group_id=None):
+    context = {
+        'model': model,
+        'ignore_auth': True,
+        'validate': False,
+        'use_cache': False
+    }
+
+    _entity_id = group_id if group_id else toolkit.config.get('ckan.site_id')
+    task = {
+        'entity_id': _entity_id,
+        'entity_type': 'group' if group_id else 'site',
+        'task_type': 'search_rebuild',
```

Comment:
the things we're actually reindexing are packages, so `entity_type="group"` and `task_type="search_rebuild"` might be confusing here

Reply:
@wardi I changed the `task_type` to be `reindex_packages` which matches the job's callback method name.